src/backend/marsha/core/views.py

```diff
@@ -57,7 +61,8 @@ def get_context_data(self):
         lti = self.get_lti()
         try:
             video = lti.get_or_create_video()
-        except LTIException:
+        except LTIException as error:
+            logger.warning("LTI Exception: %s", error)
```

Comment:
Current Marsha logging configuration will not allow `warning`log level to appear in Sentry nor in Openshift logs

Reply:
You're right! So I checked and Sentry IS in fact logging warnings. Just that it is logging them as `breadcrumbs` inside the next error... not what we want. 

It was not logging the warning in stdout because there was a bug in the logging configuration. I added a [commit](https://github.com/openfun/marsha/pull/220/commits/d4929e0227dbcb445f6841a78b3f87e5e31d7787) to fix it and the warning should now come up in the logs. I think it's enough and we don't want it in Sentry anyway.